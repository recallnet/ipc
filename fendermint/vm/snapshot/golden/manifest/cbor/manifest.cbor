a66c626c6f636b5f6865696768741affffffff6473697a651ba23692619638baae666368756e6b731a8811e90768636865636b73756d7840374230303632334643343541443935313745453937353643303037383533324632323039453231433941454130304230324235383339373838393638324434466c73746174655f706172616d73aa6a73746174655f726f6f74d82a4a00014f4c05edc2c4b8906974696d657374616d701bffffffffffffffff6f6e6574776f726b5f76657273696f6e1affffffff68626173655f6665655100e3e0cec31b8d04f4faecb1cbc89b35116b636972635f737570706c795100fe4fec75162112c09867d1e9ee57054968636861696e5f69641b000c73e07826f6606b706f7765725f7363616c65006b6170705f76657273696f6e00756372656469745f64656269745f696e74657276616c0075626c6f625f73746f726167655f63617061636974791bcbb335ee36f61e676776657273696f6e1a769695b2