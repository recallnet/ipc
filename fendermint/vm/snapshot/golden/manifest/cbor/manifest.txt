SnapshotManifest { block_height: 4294967295, size: 11688690830716484270, chunks: 2282875143, checksum: Hash::Sha256(7B00623FC45AD9517EE9756C0078532F2209E21C9AEA00B02B58397889682D4F), state_params: FvmStateParams { state_root: Cid(bafhuybpnylclrea), timestamp: Timestamp(18446744073709551615), network_version: NetworkVersion(4294967295), base_fee: TokenAmount(302902023173884324277.902684861920654609), circ_supply: TokenAmount(338038898306403012386.838857616742352201), chain_id: 3505107646215776, power_scale: 0, app_version: 0, credit_debit_interval: 0, blob_storage_capacity: 14678134907771166311 }, version: 1989580210 }